test/cmd/lfstest-gitserver.go

```diff
@@ -481,6 +497,98 @@ func storageHandler(w http.ResponseWriter, r *http.Request) {
 		}
 
 		w.WriteHeader(404)
+	case "HEAD":
+		// tus.io
+		if len(r.Header.Get("Tus-Resumable")) == 0 {
+			log.Fatal("Missing Tus-Resumable header in request")
+			w.WriteHeader(400)
+			return
+		}
+		parts := strings.Split(r.URL.Path, "/")
+		oid := parts[len(parts)-1]
+		if by, ok := largeObjects.GetIncomplete(repo, oid); ok {
+			w.Header().Set("Upload-Offset", strconv.FormatInt(int64(len(by)), 10))
+		} else {
+			w.Header().Set("Upload-Offset", "0")
+		}
+		w.WriteHeader(200)
+	case "PATCH":
+		// tus.io
+		if len(r.Header.Get("Tus-Resumable")) == 0 {
+			log.Fatal("Missing Tus-Resumable header in request")
+			w.WriteHeader(400)
+			return
+		}
+		parts := strings.Split(r.URL.Path, "/")
+		oid := parts[len(parts)-1]
+
+		offsetHdr := r.Header.Get("Upload-Offset")
```

Comment:
Sorry, this wasn't clear. I was suggesting that you inline the `r.Header.Get` into L530's `strconv.ParseInt` like:

```go
offset, err := strconv.ParseInt(r.Header.Get("Upload-Offset"), 10, 64)
```

But it was super minor anyway 😄